AEM/optimum-habana-perf.md

```diff
@@ -44,26 +45,28 @@ pip install git+https://github.com/HabanaAI/DeepSpeed.git@1.19.0
 ```
 
 
-# Tensor quantization statistics measurement
-
-## Llama2
+# Tensor quantization statisics measurement
+Users just need to do once among different models with related world size values.  
+Users will get hqt_output after this step and it will be used for fp8 run.  
+If users plan to change models for fp8 run, they need to do this step again to get relted hqt_output.   
```

Comment:
Change to "Changing models will require regenerating the contents of the hqt_output directory."

Reply:
use iGPT to improve the sentences accordingly.